internet_time.go

```diff
@@ -0,0 +1,103 @@
+package swatch
+
+import (
+	"math"
+	"time"
+)
+
+var (
+	defaultAlgorithm         = TotalSeconds
+	secondsPerBeat           = 86.4
+	nanoPerDay         int64 = 8.64e+13
+	nanoPerBeat        int64 = nanoPerDay / 1000
+	nanoPerHour        int64 = 3.6e+12
+	maxSwatchPrecision int   = 6
+)
+
+func New() *internetTime {
+	return &internetTime{getUtcTime(time.Now()), defaultAlgorithm}
+}
+
+// Alias for New
+func Now() *internetTime {
+	return New()
+}
```

Comment:
I didn't notice `NewUsing` until now.
Regardless, I'm a fan of the functional options pattern. There's a few common ways to implement it, and I'll probably suggest one in a future commit.
This would allow us to just define `swatch.New` instead of multiple constructors.
I.e.
`New()`, `New(WithAlgorithm($A))`, `New(FromTime($T))`, and can be combined (order irrelevant), `New(WithAlgorithm($A), FromTime($T))`

Reply:
Functional options in Go is a new concept to me, that'll be interesting to see. Does look quite un-go to me though